Gemfile.lock

```diff
@@ -265,12 +263,6 @@ GEM
     google-protobuf (4.29.2)
       bigdecimal
       rake (>= 13)
-    google-protobuf (4.29.2-arm64-darwin)
-      bigdecimal
-      rake (>= 13)
-    google-protobuf (4.29.2-x86_64-linux)
-      bigdecimal
-      rake (>= 13)
```

Comment:
hmmm strange that these gems for these platforms got removed 🤔

Reply:
This is odd. I try to revert this changes but `bundle install` keeps doing it.

When it fails, on [GH actions](https://github.com/rootstrap/rails_api_base/actions/runs/12509037408/job/34897961331) it says :
```
ffi-1.17.0-x86_64-linux-gnu requires ruby version < 3.4.dev, >= 2.5, which is
incompatible with the current version, 3.4.1
```

Maybe is no more needed? I couldn't find anything on google 😂

Reply:
😮 ok. Let's go ahead and if something breaks we can fix it